CMakeLists.txt

```diff
@@ -300,9 +300,6 @@ include_directories(SYSTEM ${LIBUV_INCLUDE_DIRS})
 find_package(Msgpack 1.0.0 REQUIRED)
 include_directories(SYSTEM ${MSGPACK_INCLUDE_DIRS})
 
-find_package(LuaJit REQUIRED)
-include_directories(SYSTEM ${LUAJIT_INCLUDE_DIRS})
```

Comment:
If LUAJIT_INCLUDE_DIRS is not actually needed, I guess we can remove that logic from `cmake/FindLuaJit.cmake` (in a separate commit). It could save some time in the build.